demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -323,25 +338,34 @@ export class DemoMeetingApp {
     });
   }
 
-  async authenticate(): Promise<string> {
-    let joinInfo = (await this.joinMeeting()).JoinInfo;
-    const configuration = new MeetingSessionConfiguration(joinInfo.Meeting, joinInfo.Attendee);
-    await this.initializeMeetingSession(configuration);
-    return configuration.meetingId;
+  createEmptyMeetingSessionConfiguration(): MeetingSessionConfiguration {
+    const emptyConfiguration = new MeetingSessionConfiguration();
+    emptyConfiguration.meetingId = '';
+    emptyConfiguration.credentials = new MeetingSessionCredentials();
+    emptyConfiguration.credentials.attendeeId = '';
+    emptyConfiguration.credentials.joinToken = '';
+    emptyConfiguration.urls = new MeetingSessionURLs();
+    emptyConfiguration.urls.turnControlURL = '';
+    emptyConfiguration.urls.audioHostURL = '';
+    emptyConfiguration.urls.screenViewingURL = '';
+    emptyConfiguration.urls.screenDataURL = '';
+    emptyConfiguration.urls.screenSharingURL = 'wss://localhost/';
+    emptyConfiguration.urls.signalingURL = 'wss://localhost/';
+    return emptyConfiguration;
   }
 
-  async initializeMeetingSession(configuration: MeetingSessionConfiguration): Promise<void> {
+  async initializeLoggerAndDeviceController(configuration?: MeetingSessionConfiguration): Promise<void> {
     let logger: Logger;
     const logLevel = LogLevel.INFO;
     const consoleLogger = (logger = new ConsoleLogger('SDK', logLevel));
     if (location.hostname === 'localhost' || location.hostname === '127.0.0.1') {
-      logger = consoleLogger;
+      this.logger = consoleLogger;
```

Comment:
Ridip recently added createLogStream for multi logger which we would want to do here as well